docs/fastdds/property_policies/non_consolidated_qos.rst

```diff
@@ -474,3 +474,65 @@ The different property values have the following effects on the local |DomainPar
         :start-after: <!-->TYPE_PROPAGATION_PROPERTY<-->
         :end-before: <!--><-->
         :lines: 2-4,6-17,19-20
+
+.. _property_serialize_optional_qos:
+
+Add optional QoS to EDP messages
```

Comment:
NIT: I would probably be more generic in the title of the section to something more `Adding optional QoS to Discovery data`, later on we can specify that it is in EDP. But in this way, when we add optional QoS in the participant we can extend this section.